rebar.config

```diff
@@ -217,7 +221,8 @@
    {override, jwerl, [{plugins, [rebar3_elixir, rebar3_hex]}]},
    %% GraphQL pulls geas_rebar3, which is a dev dependency
    %% geas takes some time to compile, but we do not need it
-   {override, graphql, [{plugins, [rebar3_hex]}]}
+   {override, graphql, [{plugins, [rebar3_hex]}]},
```

Comment:
Btw, can we fix this directly on the dep? https://github.com/esl/graphql-erlang? We have our own fork so it's easy to fix there.

Reply:
PR created https://github.com/esl/graphql-erlang/pull/5